### ISSUE TITLE
RORA fetcher logging improvements

### ISSUE THREAD

**Issue by redlicha** [openvstorage/alba#326]
During tests we've seen the following in the volumedriver log:
```
2016-08-12 15:00:12 602232 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/AlbaProxyClient - 000000000000058f -  info - logMessage: RDMAProxy_client(172.19.12.141, 26204)
2016-08-12 15:00:12 602526 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/AlbaProxyClient - 0000000000000590 -  info - logMessage: connecting
2016-08-12 15:00:12 605926 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/AlbaProxyClient - 0000000000000591 -  info - logMessage: RoraProxy_client(...)
2016-08-12 15:00:12 624812 +0200 - cmp02 - 26398/0x00007f48d97fa700 - volumedriverfs/(UnspecifiedComponent) - 0000000000000592 -  error - 
 Alert !!imsg has error in retval

2016-08-12 15:00:12 624988 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/(UnspecifiedComponent) - 0000000000000593 -  error - giocbp->request_->_failed is true. Error is -5
2016-08-12 15:00:12 625080 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/AlbaProxyClient - 0000000000000594 -  info - logMessage: retcode:-5
2016-08-12 15:00:12 626663 +0200 - cmp02 - 26398/0x00007f48d9ffb700 - volumedriverfs/AlbaProxyClient - 0000000000000595 -  info - logMessage: ~RDMAProxy_client
```

The "retcode:-5" message comes from the RORA fetcher and should be at `error` level and should if possible also provide information about the target (ASD) of the failed request.

**Comment by redlicha** [comment]
```
int OsdAccess::_read_osd_slices(osd_t osd, std::vector<asd_slice> &slices) {
  // ...
  int ret = aio_readv(ctx, key_vec, iocb_vec);
  if (ret == 0) {
    ret = aio_suspendv(ctx, iocb_vec, nullptr /* timeout */);
  }
  for (auto &elem : iocb_vec) {
    auto retcode = aio_return(ctx, elem);
    if (ret != 0) {
      ALBA_LOG(INFO, "retcode:" << retcode);
    }
    aio_finish(ctx, elem);
  }
```
The message should be emitted on `retcode != 0`

**Comment by domsj** [comment]
Fixed by #329
